build.sh

```diff
@@ -1,5 +1,31 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
+set -e
+
+# This script is a modification of the build for docke2aci
+# https://github.com/appc/docker2aci/blob/master/build.sh
+
+
+ORG_PATH="github.com/kinvolk"
+REPO_PATH="${ORG_PATH}/tcd"
+
+if [ ! -h gopath/src/${REPO_PATH} ]; then
+    mkdir -p gopath/src/${ORG_PATH}
+    ln -s ../../../.. gopath/src/${REPO_PATH} || exit 255
+fi
+
+export GOBIN=${PWD}/bin
+export GOPATH=${PWD}/gopath:${PWD}/Godeps/_workspace
+
+eval $(go env)
+go env
+go get -d
```

Comment:
As said previously "go get" should be removed with the vendoring.

And I guess "go env" was just debugging. The important one is the one with eval.  It can be removed.